service/controller/cert.go

```diff
@@ -133,6 +133,7 @@ func NewCert(config CertConfig) (*Cert, error) {
 	var newInformer *informer.Informer
 	{
 		c := informer.Config{
+			Logger:  config.Logger,
```

Comment:
This is the new dependency added, so we can have some logs while collecting metrics.